README.md

```diff
@@ -154,15 +199,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+INSERT INTO persons (name, age) VALUES ('Whitney Purdum', 30);
```

Comment:
As the question stated, we were looking for the ruby code which creates a new person:
```ruby
whitney = Person.create(name: "Whitney", age: 30)
```